GameData/KerbalismConfig/System/Parts.cfg

```diff
@@ -843,6 +843,61 @@
   }
 }
 
+//Soyuz MS
+//Descent module
+@PART[ROC-SoyuzMSCapsule]:NEEDS[ProfileRealismOverhaul]:AFTER[RealismOverhaul]
+{
+  %capsuleScrubbers = true
+  %scrubberCapacity = 1.1
+  %capsuleOrigin = USSR
+
+  @MODULE[ModuleCommand]
+  {
+    @RESOURCE[ElectricCharge]
+    {
+      @rate = 0.650  //Remove Climatization and LS power draw
+    }
+  }
+
+  @MODULE[ProcessController],*
+  {
+    @capacity *= #$/CrewCapacity$
+  }
+}
+
+//Orbital module
+@PART[ROC-SoyuzMSOM]:NEEDS[ProfileRealismOverhaul]:AFTER[RealismOverhaul]
+{
+  %capsuleScrubbers = true
+  // don't add the generic scrubbers since the main ones are in the descent module
+
+  MODULE
+  {
+    name = ProcessController
+    resource = _Scrubber
+    title = LiOH Scrubber (Auxiliary)
+    capacity = 0.2
+    desc =  A Lithium Hydroxide scrubber. <b>Only absorbs 20% of the CO2 produced by the crew!</b>
+    toggle = true
+    running = true
+  }
+
+  MODULE
+  {
+    name = ProcessController
+    resource = _PressureControlOxygen
+    title = O2 Pressure Controller
+    desc = Use <b>Oxygen</b> to maintain the internal atmosphere at a comfortable pressure.
+    capacity = 1.67
+    running = true
+  }
+
+  @MODULE[ProcessController],*
+  {
+    @capacity *= 3
+  }
+}
+
```

Comment:
All these patches are duplicated, you should instead add the part name into the already existing Soyuz patches

Reply:
Done